frontend/app/(footer)/event/create/page.tsx

```diff
@@ -204,7 +218,12 @@ export default function CreateEvent() {
                 </button>
               )}
               {!isLastStep && (
-                <button type="submit" className="border border-black py-1.5 px-7 rounded-lg ml-auto lg:mr-2">
+                <button
+                  type="submit"
+                  className={`border border-black py-1.5 px-7 rounded-lg ml-auto lg:mr-2 ${
+                    hasError ? "opacity-50 cursor-not-allowed" : "cursor-pointer"
```

Comment:
we need some service layer validation for this as well. A simple cursor-not-allowed wont cut it, sadge

Reply:
:(

Reply:
This does not prevent the clicking of this button. We will need some more protection on this button.

Reply:
there is a disabled part below the code snippet which ive added which stops you from clicking feel free to run it and test